index.html

```diff
@@ -321,19 +322,218 @@ <h4 class="pricing-table__subtitle-secondary my-0 pb-1">Subtitle second line goe
                 </div>
                 <div class="ad-block__text text-justify pt-1 pb-9 pt-xs-1 pb-xs-9">
                     Lorem ipsum dolor sit amet, consectetur adipisicing elit,
-                     sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-                      Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
-                       Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
-                        Excepteur sint occaecat cupidatat non proident,
-                         sunt in culpa qui officia deserunt mollit anim id est laborum.
-                          Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam,
-                           eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam 
+                    sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+                    Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+                    consequat.
+                    Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+                    pariatur.
+                    Excepteur sint occaecat cupidatat non proident,
+                    sunt in culpa qui officia deserunt mollit anim id est laborum.
+                    Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium,
+                    totam rem aperiam,
+                    eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo.
+                    Nemo enim ipsam
                 </div>
                 <div class="ad-block__button pb-10 pt-3 pt-xs-3 pb-xs-6">
                     <button class="ad-block__button--purchase px-4 py-2">Purchase now</button>
                 </div>
             </div>
         </div>
     </div>
+
+    <!-- Footer -->
```

Comment:
Cái này đang không hợp lý, nó đang là slider mà em để bố cục nó không được thống nhất qua các màn hình. Nhìn nó cũng không được cân đối, tự nhiên lẻ ra 1 ô nhảy xuống dưới. 1 là em thu nhỏ phóng to ô vuông cho vừa, 2 là tăng kích thước giữa các ô nếu không gian quá lớn hoặc kết hợp cả 2. Làm sao nó không bị lệch hẳn bố cục nữa nhá
![image](https://github.com/user-attachments/assets/65007b59-23f7-4ec6-9381-726eccc69852)